commit 7ec5f9465e732e668d0dc94ac078ba68056d6d0a
Author: Mike FABIAN <mfabian@redhat.com>
Date:   Mon Feb 19 21:59:30 2018 +0100

    Add missing “reorder-end” in LC_COLLATE of et_EE [BZ #22517]
    
            [BZ #22517]
            * localedata/locales/et_EE (LC_COLLATE): add missing “reorder-end”

diff --git a/localedata/locales/et_EE b/localedata/locales/et_EE
index 9cb55b568f..bab7493c98 100644
--- a/localedata/locales/et_EE
+++ b/localedata/locales/et_EE
@@ -103,6 +103,8 @@ reorder-after <w>
 <U00FC> <u-diaresis>;<BAS>;<MIN>;IGNORE % ü
 <U00DC> <u-diaresis>;<BAS>;<CAP>;IGNORE % Ü
 
+reorder-end
+
 END LC_COLLATE
 
 LC_CTYPE
